draft-ietf-tls-sni-encryption.md

```diff
@@ -164,23 +195,25 @@ see (#replayattack) for details.
 The decoupling of IP addresses and server names, deployment
 of DNS privacy, and protection of server certificate transmissions
 all contribute to user privacy in the face of an [@?RFC3552]-style
-adversary. Encrypting the SNI now will complete this push for privacy and 
+adversary. Encrypting the SNI completes this push for privacy and 
 make it harder to censor or otherwise provide differential treatment to 
 specific internet services.
 
 ## End-to-end alternatives {#end-to-end}
 
-Deploying SNI encryption will help thwart most of the "unanticipated" SNI usages
-described in (#snileak), including censorship and pervasive surveillance. It will
-also thwart functions that are sometimes described as legitimate. Most of
+Deploying SNI encryption helps thwart most of the unanticipated SNI usages
+described in (#snileak), including censorship and pervasive surveillance. It
+will break or reduce the efficacy of the operational practices and
+techniques implemented in middle-boxes as described in (#snileak). Most of
```

Comment:
We are referenceing ```#snileak``` in two adjacent sentences, which may be overkill (but may not, if there's not a good way to link the concepts together).

Reply:
Hopefully this works.

Reply:
It looks okay, thanks.  There's maybe something funny with how "most of the unanticipated SNI usages" leaves us hanging until we get to the reference, but I think we have had enough coverage of those unanticipated usages already that it should be okay.